transformer_engine/common/util/math.h

```diff
@@ -75,7 +75,7 @@ __device__ inline OType silu(const IType val, const Empty& e) {
 }
 
 template <typename OType, typename IType>
-__device__ inline OType oss_silu(const IType val, const GptOssParam& p) {
+__device__ inline OType oss_silu(const IType val, const ClampedSwiGLUParam& p) {
```

Comment:
Nit: `clamped_silu` and `clamped_dsilu` would be more consistent with the naming scheme.
```suggestion
__device__ inline OType oss_silu(const IType val, const ClampedSwiGLUParam& p) {
```